indigo/distribution.yaml

```diff
@@ -11131,6 +11131,25 @@ repositories:
       url: https://github.com/g/roboteq.git
       version: master
     status: maintained
+  roboteq_diff_driver:
+    doc:
+      depends:
+      - serial
```

Comment:
I don't think this is necessary since `serial` is a released package and your package declares a dependency on it.

Same below.

Reply:
I see. Shall I remove it and resubmit, or is it benign?

Reply:
You can simply remove the line and commit to the branch of this PR.

Reply:
Removed both depends.